src/api/app/views/webui2/webui/project/_breadcrumb_items.html.haml

```diff
@@ -5,23 +5,24 @@
 - else
   %li.breadcrumb-item
     = link_to 'Projects', projects_path
-  %li.breadcrumb-item
-    = link_to @project, project_show_path(@project)
-  - if current_page?(project_show_path(@project))
-    %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Overview
-  - elsif current_page?(project_monitor_path(@project))
-    %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Monitor
-  - elsif current_page?(project_requests_path(@project))
-    %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Requests
-  - elsif current_page?(project_users_path(@project))
-    %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Users
-  - elsif current_page?(project_config_path(@project))
-    %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Configuration
-  - elsif current_page?(project_status_path(@project))
-    %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-      Status
+  - if @project
```

Comment:
why this if?

Reply:
@hennevogel 
> why this if?

Because sometimes this is called without having a project. I think instead adding an if, we should ensure we only call the project breadcrum if we have a project. :thinking:

Reply:
@hennevogel, @Ana06:

And what about routes like `/project/list`, `/project/list_all`, ... that belong to a the project controller, but doesn't have a project? What do you both propose instead for rendering a breadcrumb with this kind of routes?

Reply:
to have a `id` with that route so that it is also shown in the breadcrumb

Reply:
Ok. Modified. Can you have a look again?